Altis_Life.Altis/dialog/function/fn_garageLBChange.sqf

```diff
@@ -7,29 +7,28 @@
     Can't be bothered to answer it.. Already deleted it by accident..
 */
 disableSerialization;
-private ["_control","_index","_className","_classNameLife","_dataArr","_vehicleColor","_vehicleInfo","_trunkSpace","_sellPrice","_retrievePrice","_sellMultiplier","_price","_storageFee","_purchasePrice"];
-_control = _this select 0;
-_index = _this select 1;
+private _control = _this select 0;
+private _index = _this select 1;
 
 //Fetch some information.
-_dataArr = CONTROL_DATAI(_control,_index);
+private _dataArr = CONTROL_DATAI(_control,_index);
 _dataArr = call compile format ["%1",_dataArr];
-_className = (_dataArr select 0);
-_classNameLife = _className;
+private _className = (_dataArr select 0);
+private _classNameLife = _className;
 
 if (!isClass (missionConfigFile >> "LifeCfgVehicles" >> _classNameLife)) then {
     _classNameLife = "Default"; //Use Default class if it doesn't exist
     diag_log format ["%1: LifeCfgVehicles class doesn't exist",_className];
 };
 
-_vehicleColor = ((M_CONFIG(getArray,"LifeCfgVehicles",_classNameLife,"textures") select (_dataArr select 1)) select 0);
+private _vehicleColor = ((M_CONFIG(getArray,"LifeCfgVehicles",_classNameLife,"textures") select (_dataArr select 1)) select 0);
```

Comment:
Could we maybe change this to use params or param for more readability? With types provided etc